### ISSUE TITLE
Display exported path

### ISSUE THREAD

**Issue by codespear** [qjebbs/vscode-plantuml#44]
It would be helpful to see the full path of the exported file(s) on the export complete notification message.

**Comment by qjebbs** [comment]
In what scenario you want see the paths? Exported files are organized exactly the same with how you organize your code, it's predictable.

**Comment by codespear** [comment]
You have a point: and a good question. I think the scenario I experienced is not a common one.  

One factor is the `plantuml.exportOutDirName` setting.  If you have a user setting to `./` and in one of your older projects you had an override for the value (say to `./diagrams`) things do  become a little less predictable (because you forget about the settings).
   
I have many diagrams in one file; the name of the output file depends on the name of the source file and on the title of the diagram. 

In my case a second factor that added a little confusion is that I moved the diagram source file  to a different sub-folder, thereby (unintentionally) I changed  the output folder for the images.  It took a second or two to figure out what as going on; so I posted this suggestion.

Btw: _great plugin!_
